cmd/analyze/main.go

```diff
@@ -100,14 +101,34 @@ func dynamicAnalysis(pkg *pkgmanager.Pkg) {
 		}
 	}
 
+	fileWriteDataUploadStart := time.Now()
+
 	if *uploadFileWriteInfo != "" {
 		bucket, path := parseBucketPath(*uploadFileWriteInfo)
-		err := resultstore.New(bucket, resultstore.BasePath(path)).Save(ctx, pkg, results.FileWrites)
+		err := resultstore.New(bucket, resultstore.BasePath(path)).Save(ctx, pkg, results.FileWritesSummary)
```

Comment:
Instead of calling `resultstore.New(...)` twice, you can save the result and re-use it here and bellow.

e.g.

```go
rs := resultstore.New(bucket, ...)
err := rs.Save(...)
...
err := rs.SaveWriteBufferZip(...)
...